src/pass.js

```diff
@@ -101,7 +101,8 @@ export class Pass {
   cancel() {
     if (this.scheduled_ != -1) {
       timer.cancel(this.scheduled_);
-      this.scheduled_ = -1;
     }
+    this.scheduled_ = -1;
+    this.nextTime_ = 0;
```

Comment:
Does this mean there was a bug? Is a unit test needed to verify this? Asking since this is a widely used class.